src/main/openapi/ds-datahandler-openapi_v1.yaml

```diff
@@ -320,6 +318,26 @@ components:
                             [Preservica](https://preservica.com/) XIP content of the record
           example: 'direct'
           default: 'direct'
+        dateStampFormat:
+          type: string
+          enum:  ['day', 'date']
+          description: |
+            Special handling of record. Possible value are
+            * `day`:     Only use format yyyy-MM-dd as from/until parameters
+            * `date`:    Use UTC format yyyy-MM-ddThh:mm:ssZ as from/until parameters. This is default.                        
+          example: 'date'
+          default: 'date'
+        day_only:
+          type: boolean          
+          description: 'OAI harvest will fetch by day using from until. This is required for OAI servers that does not scale'                        
+          example: true
+          default: false        
+        start_day: 
+          type: string
+          description: 'Date of format yyyy-MM-dd. Only used if day_only attribute is true'                        
+          example: '2020-01-01'
```

Comment:
Please use either camelCase **or** under_score for multi word handling.

Reply:
done, also change oaiTargets (list name)

Reply:
Seems like I was unclear here: `dateStampFormat`, `day_only` and `start_day` in the code above uses 2 different ways of handling multi word.

Or maybe you forgot a commit?

Reply:
Seems I forgot and only fixed it in the yaml-file and not also openAPI.
Fixed now

Reply:
Also in behavior yaml.